### DISCUSSION TITLE
Parallel execution

### DISCUSSION THREAD

**Discussion by ttscoff** [ttscoff/bunch-docs#15] [Q&A] [answered]
The latest updates have a new feature called parallel execution. You can find out [more about it here](/bunch-beta/docs/bunch-files/sequence/). It's not actually parallel from a compsci perspective, but does execute all items asynchronously, leading to everything basically launching at once.

Running in parallel does have the side effect of things appearing to happen out of order, as one process isn't waiting for its predecessor to complete before launching. I would guess that in about 50% of Bunch scenarios, this is fine, and it leads to _very_ fast launches. But because in the other 50% of scenarios it's not desirable, this option can be turned on and off both globally and per-file. 

If everyone would give it a shot and let me know what issues you run into, I'd appreciate it. The easiest way to get started is just to go to Preferences and enable "Launch Items in Parallel" and see what happens with your commonly-used Bunches.

**Comment by geofftaylor** [comment, marked as answer]
Did this option disappear in beta 69? I swear I had it enabled in an earlier beta, but I just noticed that it's missing from the preferences.

**Reply by ttscoff** [reply]
Oh that's weird. Entirely unintentional, will look into it.

**Reply by ttscoff** [reply]
Ok, update pushed. In addition to this fix, a couple of new features worth reading about!

**Reply by geofftaylor** [reply]
I see it now. I like the multiple choice dialogs. Nice feature!
